packages/api-react/src/services/wallet.ts

```diff
@@ -1396,6 +1396,31 @@ export const walletApi = apiWithTag.injectEndpoints({
     }),
 
     verifySignature: mutation(build, WalletService, 'verifySignature'),
+
+    getVC: build.query<string, { vcId: string }>({
```

Comment:
Can you update this to use the newer `query(build, VC, 'getVC')` syntax? Take a look at `getCATName` as an example.